ui/src/dashboards/containers/DashboardPage.js

```diff
@@ -200,8 +201,19 @@ class DashboardPage extends Component {
     this.setState({zoomedTimeRange: {zoomedLower, zoomedUpper}})
   }
 
+  getActiveDashboard() {
+    const {params: {dashboardID}, dashboards} = this.props
+    return dashboards.find(d => d.id === +dashboardID)
+  }
+
```

Comment:
This function isn't used anywhere anymore in this component.  Let's ✂️  it.

Reply:
good eye!